src/experimental-layers/src/mesh-layer/mesh-layer.js

```diff
@@ -66,6 +66,23 @@ function getTextureFromData(gl, data, opts) {
   return new Texture2D(gl, Object.assign({data}, opts));
 }
 
+/*
+ * Convert mesh data into geometry
+ * @returns {Geometry} geometry
+ */
+function getGeometry(data) {
+  if (data instanceof Geometry) {
+    assert(data.attributes.positions);
+    return data;
+  } else if (data.positions) {
+    return new Geometry({
```

Comment:
This second path seems like a somewhat arbitrary choice and should preferable at least be documented. Maybe time to start a markdown page for this layer? Looks like it is being groomed for inclusion.